nftest.yml

```diff
@@ -16,10 +16,10 @@ cases:
     asserts:
       - actual: generate-SQC-BAM-*/TWGSAMIN000001/SAMtools-*/output/SAMtools-*_TWGSAMIN_HG002.N_stats.txt
         expect: /hot/software/pipeline/pipeline-generate-SQC-BAM/Nextflow/development/test-output/SAMtools-1.18_TWGSAMIN_HG002.N_stats.txt
```

Comment:
I'd recommend to update the version in `expect` file as well so the expected file always represents the updated SAMtools version.

Reply:
Thanks @Faizal-Eeman but I think we decided that unless the relevant output has changed there was no need to keep updating the expect files or filenames.

Reply:
Generally yes, to avoid having to constantly make updates and maintain basically the same file for multiple versions, the expected files don't need to be updated until an actual change in the expected file arises